packages/core/logging/core-logging-common-internal/src/layouts/conversions/message.test.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { LogLevel, LogRecord } from '@kbn/logging';
+import { MessageConversion } from './message';
+
+const baseRecord: LogRecord = {
+  pid: 1,
+  timestamp: new Date(),
+  level: LogLevel.Info,
+  context: '',
```

Comment:
`context` is also a string, but I couldn't find the `pattern` layout exposing it, so I would expect it being logged only in the JSON layout, in which case, the escape codes are not working.